recipe/meta.yaml

```diff
@@ -21,15 +21,15 @@ requirements:
     - python
     - setuptools
     - wheel
+    - cython >=0.28.4,<4
   run:
     - ply >=3.4,<4.0
     - python
-    - six 1.15.*
+    - six >=1.15,<2.dev0
```

Comment:
should this not be `2.0a0`?

Reply:
I'm not sure the difference between those two, but maybe `2.*` would work best and catch all cases?

Reply:
that was grayskull output, but it's correct.
https://github.com/conda/conda/blob/main/conda/models/version.py#L110
_'dev' versions are smaller than all corresponding versions of other types_